docs/providers/aws/guide/credentials.md

```diff
@@ -184,6 +184,14 @@ You can always specify the profile which should be used via the `aws-profile` op
 serverless deploy --aws-profile devProfile
 ```
 
+##### Disable aws credential validation
+
+By default serverless validates the aws credentials. This can be deactivated, if you would like to use features such as ECSCredentials or TokenFileWebIdentityCredentials where the serverless validation fails.
```

Comment:
In most of this document, aws was written with uppercase `AWS`. I think it could be a good idea to follow the same pattern.